docs/catalog/ai/energy-efficent-models.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - machine-learning
+ - role:data-scientist
+ - size:small
+---
+
+# Evaluate and use energy-efficient AI/ML models
+
+## Description
+
+Large ML and deep learning network models have a significant carbon footprint. Evaluate and use alternative energy-efficient models that provide similar functionality.
+
+
+## Solution
+Evaluate energy-efficient AI/ML models for development and inference. For instance, use DistilBERT which provides similar functionality as the BERT model, which runs 60% faster while preserving 97% of BERT's performance. GPT-Neo 125M provides balanced results for energy consumption and accuracy as compared to GPT-J 6B or GPT Neo 2.7B models 
+
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+For the SCI equation, using energy-efficient AI/ML models would impact the following:
+
+- 'E':  Having an energy-efficient AI/ML model would provide efficient resource utilization and reduces the energy consumption of the compute resources and consequently, the E number should decrease.
+
+## Assumptions
+None 
```

Comment:
Are there scenarios where BERT or DistilBERT models wouldn't be available?

Reply:
Yes, based on the model section, you would pick up a pre-trained model. BERT or DistilBERT are just of examples. Pls, let me know what we are looking for in Assumption.